protos/action/action.proto

```diff
@@ -177,6 +183,26 @@ message GotoLocationResponse {
     ActionResult action_result = 1;
 }
 
+enum OrbitYawBehavior {
+        ORBIT_YAW_BEHAVIOUR_HOLD_FRONT_TO_CIRCLE_CENTER = 0;
+        ORBIT_YAW_BEHAVIOUR_HOLD_INITIAL_HEADING = 1;
+        ORBIT_YAW_BEHAVIOUR_UNCONTROLLED = 2;
+        ORBIT_YAW_BEHAVIOUR_HOLD_FRONT_TANGENT_TO_CIRCLE = 3;
+        ORBIT_YAW_BEHAVIOUR_RC_CONTROLLED = 4;
+}
+    
+message DoOrbitRequest {    
+    float radius_m = 1; // Radius of circle (in meters)
+    float velocity_ms = 2; // Tangential velocity (in m/s)
+    OrbitYawBehavior yaw_behavior = 3; // Yaw behavior of vehicle (ORBIT_YAW_BEHAVIOUR)
+    double latitude_x = 5; // Center point latitude (no mav_frame) / X coord to mav_frame. NAN: current
```

Comment:
I am not really sure what this means, still. From where did you copy that?

Usually we have `latitude_deg`, I guess the x would be in meters (like `x_m` maybe?). The thing is that I am not sure if MAVSDK supports flying without having a GPS fix (yet). Did you try using `x` instead of `latitude`?

If it only works with a latitude, I would change that for `latitude_deg` and not expose the `xyz` ones.

What do you think?

Reply:
Its from the MAVLINK messages for MAV_CMD_DO_ORBIT param descriptions. I believe is allows for specifying a location to orbit. If no mavframe is given and the value is treated as a GPS location. If nan is given the it just uses the position that would be at the center of the current radius. I'm guess since it doesn't have a parameter for mavframe that is an option for the future? I would be fine with removing all the options for position and just keeping it simple so that when the command is given it just starts the orbit based on the center point to the radius.

Reply:
Intuitively, I would expect to set an orbit with a center and a radius. That's what we have here, so I would keep it. Except that if we don't know if that `xyz` thing works, I would suggest we only go with the GPS coordinates. That's what you are testing, isn't it?

So something like:

```
double latitude_deg = 5; // COMMENT
double longitude_deg = 6; // COMMENT
double absolute_altitude_m = 7; // COMMENT
```

Would that work for you?

Reply:
That works for me. I did test inputting NAN as well and that worked as well. It just starts the orbit at the current position based on the set radius.